src/components/UploadFileDialog.tsx

```diff
@@ -128,16 +137,24 @@ export class UploadFileDialog extends React.Component<UploadFileDialogProps, {
           <Button
             icon={<GoFile />}
             label="Upload"
-            title="Upload New File"
-            isDisabled={!this.state.files.length || !!this._nameError()}
+            title="Upload files"
+            onClick={() => {
+              document.getElementById("file-upload-input").click();

```

Comment:
This is a React anti-pattern. I think what you need to do, is add a `ref` on the FileUploadInput component, then here you can say something like `this.fileUploadInput.open()`.